docs/user/experiments-api.md

```diff
@@ -25,8 +25,8 @@ Glean.setExperimentActive(
 Glean.setExperimentInactive("blue-button-effective")
 ```
 
-> **Important**: Experiment IDs and branch don't need to be pre-defined in the Glean SDK registry files.
-Please also note that the `extra` map is non-nested arbitrary `String` to `String` map.
+> **Important**: Experiment IDs and branches don't need to be pre-defined in the Glean SDK registry files.
+Please also note that the `extra` map is a non-nested arbitrary `String` to `String` map.
```

Comment:
Can we call out the restrictions to the `extra` field keys and size?

Reply:
Yeah -- that's included below, but I'll call that out here.